### ISSUE TITLE
BYU - 6. Taxonomy (Classifications)

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/data-migration#104]
Read the [Taxonomy](http://handbook.arctosdb.org/documentation/taxonomy.html) documentation

For any names that were added in the previous data migration step, there will not be a related classification in any of the local Arctos taxonomy sources. For a small number of new names, these can be created manually by following the instructions in [How To Create Taxa](http://handbook.arctosdb.org/how_to/How-to-Create-Taxa.html). To create a large number of classifications, use the [Bulkload Taxonomy Classifications Tool](http://arctos.database.museum/tools/BulkloadClassification.cfm)

**Comment by byuherpetology** [comment]
If you bulkload taxon names, can you still use the clone classification to add the higher level classification for the taxon?  I am wondering if there are only a few taxon names to add if it is easier to just add them manually rather than bulkloading?

Other than that, I have read the information and I think I understand.

**Comment by dustymc** [comment]
Yes names and classifications may/must be added independently.

**Comment by Jegelewicz** [comment]
> If you bulkload taxon names, can you still use the clone classification to add the higher level classification for the taxon?

Yes! Just find what you want to clone and use the "Clone classification into existing name" function. (Just be prepared with the name you want to clone into!)

![image](https://user-images.githubusercontent.com/5725767/66600566-01714180-eb63-11e9-9deb-48519c74cc0b.png)

> I am wondering if there are only a few taxon names to add if it is easier to just add them manually rather than bulkloading?

I think so and this is what I usually do.

**Comment by Jegelewicz** [comment]
I have granted you access to manage taxonomy. You will probably need to log out and log back in before you can actually do anything with it.

Now, want to join the taxonomy committee? :-)

[Taxonomy Committee Project](https://github.com/ArctosDB/arctos/projects/12)

[Taxonomy Committee](https://github.com/orgs/ArctosDB/teams/taxonomy)

**Comment by byuherpetology** [comment]
Thank you, that's working now.  

I took a look at the taxonomy committee and project, honestly I'm not much of a taxonomist and it looks a bit out of my league.  I doubt I would have much to add at this point but I'm happy to help out if I can.

**Comment by Jegelewicz** [comment]
Feel free to join one of our meetings - the next one is on Wednesday, October 16 at 2PM mountain time. We meet in the [Zoom Room](https://berkeley.zoom.us/j/363481382). I think you will find that none of us (except Derek) are taxonomists and we are just trying to make taxonomy work better for everyone!
